bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/util/Constants.java

```diff
@@ -22,6 +22,7 @@ public class Constants {
     public static final String CLASS_SUFFIX = ".class";
     public   static final String PROFILE_ANALYZER = "io/ballerina/runtime/profiler/runtime/ProfileAnalyzer";
     public   static final String GET_INSTANCE_DESCRIPTOR = "()L" + PROFILE_ANALYZER + ";";
```

Comment:
```suggestion
    public static final String PROFILE_ANALYZER = "io/ballerina/runtime/profiler/runtime/ProfileAnalyzer";
    public static final String GET_INSTANCE_DESCRIPTOR = "()L" + PROFILE_ANALYZER + ";";
```
Minor suggestion, we can change this later

Reply:
I'll address this in the issue https://github.com/ballerina-platform/ballerina-lang/issues/41298